inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/sink/ElasticsearchSinkServiceTest.java

```diff
@@ -97,4 +109,16 @@ public void testGetAndUpdate() {
         deleteSink(sinkId);
     }
 
+    private void prepareEsDataNode(String dataNodeName) {
+        ElasticsearchDataNodeRequest request = new ElasticsearchDataNodeRequest();
+        request.setName(dataNodeName);
+        request.setType(DataNodeType.ELASTICSEARCH);
+        request.setUrl("http://127.0.0.1:9200;http://127.0.0.1:9200");
```

Comment:
http://127.0.0.1:9200;http://127.0.0.1:9200
->
http://127.0.0.1:9200;http://127.0.0.1:9300

It's better to use a different URL.

Reply:
Okay, I will improve it